website/src/components/Showcase/Showcase.js

```diff
@@ -5,8 +5,19 @@ import cardList from '@site/showcase.json';
 import styles from './Showcase.module.css';
 
 function Showcase() {
-  const cards = cardList.map((props) => <Card key={props.title} {...props} />);
+  const cards = _makeShuffledCardList().map(_makeCard)
+  return _makeCardsContainer(cards);
+}
+
+const _makeShuffledCardList = () => {
+  return cardList.sort(() => 0.5 - Math.random());
+}
```

Comment:
```suggestion
const _makeShuffledCardList = () => (cardList.sort(() => 0.5 - Math.random()));
```

Reply:
I think we should keep the Wix ones at the top:
1. Would be weird to have it somewhere in the middle
2. Need to stay adjacent, as a group

Reply:
OK, I agree on both points.. Done